xml/geo_booth_i.xml

```diff
@@ -64,6 +64,75 @@
    parties.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec.ha.geo.booth.ticket-types">
+  <title>Normal versus Manual Tickets</title>
```

Comment:
Potentially manual tickets could be more adopted by users. So I wouldn't imply it's abnormal ;-) How about drop the term "normal tickets" and say "Automatic versus Manual Tickets" ?

Reply:
Haha, yes. :laughing: The term "automatic" is much better, I've used this.